.github/workflows/scheduled-release.yml

```diff
@@ -0,0 +1,154 @@
+name: Schedule a release and image
+on:
+  workflow_dispatch:
+  schedule:
+     - cron: "0 0 * * 3"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: acn-dgopa/altinn-access-management-frontend
```

Comment:
Some references to your (acn-dgopa) fork of the repo which need to be updated

Reply:
Good catch :)